### ISSUE TITLE
1.6.5 r3 lighting issue.

### ISSUE THREAD

**Issue by RemusMar** [sunag/sea3d#5]
Hi,

The negative scale bug seems to be fixed, but there is another one.

1) the previous sea3d.min with my workaround:
http://necromanthus.com/Test/html5/SMC.html 
Everything looks ok.

2) the new sea3d.min without any workaround:
http://necromanthus.com/Test/html5/SMC_new.html
The lighting (on buildings) is flipped on Z (Y in 3DS Max).

cheers

**Comment by RemusMar** [comment]
For skinned meshes everything is ok:
Old: http://necromanthus.com/Test/html5/Lara.html 
New: http://necromanthus.com/Test/html5/Lara_new.html 

But for static meshes ... see the above lighting bug.

**Comment by RemusMar** [comment]
If the light is loaded from 3DS Max, everything looks ok:
http://necromanthus.com/Test/html5/SMC_new2.html 

The bug: (X,Y) in 3DS Max become (-X,-Z) in THREE.
It should become (X,Z).
cheers

**Comment by RemusMar** [comment]
sunag (Jean Carlo),

I've done more testings and that "Y to -Z" issue is a minor one.
All the other 3DS Max exporters generate the models with PI/2 rotation on the X axis.
 It can be corrected with a single JavaScript line, so let's forget about it.
Overall 1.6.5 r3 is well done.
Congrats.

A few thoughts:

1) please investigate why Spot Light is exported as Directional Light.

2) onProgress is worthless.
Please rename "onDownloadProgress" with "onProgress" and
"position" with "loaded"
"length" with "total"
It will unify the SEA3DLoader with the other THREE loaders.

all the best

**Comment by sunag** [comment]
Hi @RemusMar 

Thank you for tests! I renamed position and length. Try now using onProgress can get the LOAD_PROGRESS for complex models in low cpu processing and DOWNLOAD_PROGRESS for network data acquisition.

```javascript
var loader = new SEA3D.Loader()

loader.onProgress = function( e ) {
	if ( e.type == THREE.SEA3D.Event.DOWNLOAD_PROGRESS ) {
		// network progress
	} else if ( e.type == THREE.SEA3D.Event.LOAD_PROGRESS ) {
		// local progress
	}
}
```
If using onProgress not use in combination with onDownloadProgress or onLoadProgress.

I made a brief update on the site with some new features.
http://developers.poonya.com/forum/viewtopic.php?t=55

Thanks,
Cheers.
